plugin.xml

```diff
@@ -17,7 +17,8 @@
     <platform name="ios">
       <header-file src="src/ios/Intercom.h" />
       <source-file src="src/ios/libIntercom.a" framework="true" compiler-flags="-ObjC" />
```

Comment:
No problem :smile: I must admit my cordova knowledge is pretty thin. I would suggest replacing `libIntercom.a` with `Intercom.framework` and using these lines instead:

```
<header-file src="src/ios/Intercom.framework/Versions/A/Headers/Intercom.h" />
<source-file src="src/ios/Intercom.framework/Versions/A/Intercom" framework="true" />
```

If you like I can open a PR with this change so you don't have to.